### PR TITLE
Components

### PR DESCRIPTION
**Pull request by Donard97**
**I created this PR to check if our project was as requested.**

_This is the first project on React module. I created a calculator._ 

### Project Requirement:

- Deleted all files of CRA ✔️ 
- Created a directory called components with files Calculator.js ✔️ 
- Added components to match the design ✔️ 
- Used class-based components  ✔️ 
- Import the Calculator to app.js ✔️ 

<hr>

_Feel free to review my project and let me know if there are any essential changes 💪 ._

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```
Comment (henatan99) [inline src/App.js]: - [x] The App.js component should be class-based as per the project's requirement. The objective of this milestone it to practice class-based components. Kindly refactor your component to a class-based component. :pray:

File: src/index.js [hunk 2]
```diff
@@ -1,17 +1,12 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+// eslint-disable-next-line import/extensions
+import App from './app';
```
Comment (henatan99) [inline src/index.js]: - [x] You gave the wrong file name here for the path. Kindly fix it. :pray: 

Screenshot from 2021-09-22 01-26-05

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Donard97 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Donard97) [reply src/App.js]: Thank you for reviewing my code. I changed the component to class-based.

File: src/index.js (hunk 2 above)
Comment (Donard97) [reply src/index.js]: Thank you for reviewing my code. I fixed the file name.

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @Donard97 , 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**